netgrasp/utils/mac.py

```diff
@@ -0,0 +1,255 @@
+from __future__ import print_function
+
+import os
+import re
+import datetime
+
+TEST_WSDATA = False
+TEST_WSDATA_FILENAME = '~/.netgrasp.mac.wsdata'
+
+DB_LOOKUP_TABLE_NAME = 'macvendor'
+DB_LOOKUP_CONFIG_SOURCE_KEY = 'maclookup_source'
+DB_LOOKUP_CONFIG_UPDATED_KEY = 'maclookup_updated'
+
+LOOKUP_SOURCES = ['local', 'remote']
+
+DEFAULT_LOOKUP_SOURCE = 'remote'
+DEFAULT_LOOKUP_UPDATED = datetime.datetime(1970, 1, 1)
+DEFAULT_LOOKUP_CONFIG = dict(source=DEFAULT_LOOKUP_SOURCE, updated=DEFAULT_LOOKUP_UPDATED)
+
+WIRESHARK_ATTRIBUTION = """\
+data provided by: Wireshark - Network traffic analyzer
+                  Copyright 1998 Gerald Combs
+                  Wireshark is free software; licensed under the terms of
+                  the GNU General Public License.\
+"""
+
+
```

Comment:
the rest of the files have a single extra carriage return between functions; let's maintain the same style in this PR (repeated throughout this file)